### PR TITLE
Minecraft 1.21.4

### PR DESCRIPTION
**Pull request by RealBauHD**

### CODE REVIEW

**Review by kennytv — APPROVED** [review]
Should be fine:tm:

**Comment by kennytv** [comment]
Actually the getter is kind of ugly

File: proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/UpsertPlayerInfoPacket.java [hunk 1]
```diff
@@ -256,6 +262,10 @@ public ComponentHolder getDisplayName() {
       return displayName;
     }
 
+    public boolean getShowHat() {
```
Comment (kennytv) [inline proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/UpsertPlayerInfoPacket.java]:
```suggestion
    public boolean isShowHat() {
```
Someone can emergecy change this later if needs be

Comment (RealBauHD) [reply proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/UpsertPlayerInfoPacket.java]: Oh yes IntelliJ auto-generated getter moment, thanks Kenny

**Pull request merged**